makefiles/tools/programmer.inc.mk

```
@@ -8,7 +8,11 @@ endif
 # Don't use the programmer wrapper for the CI (where speed and verbose output
 # are important)
 ifneq (1,$(RIOT_CI_BUILD))
-  USE_PROGRAMMER_WRAPPER_SCRIPT ?= 1
+  ifneq (,$(filter pyocd,$(PROGRAMMER)))
```

I'd suggest
```suggestion
  ifneq (,$(filter $(PROGRAMMER_WRAPPER_BLACKLIST),$(PROGRAMMER)))
```
and
```
PROGRAMMER_WRAPPER_BLACKLIST = pyocd
```
for future proofing and more self explanatory code

---

Done